Lib/test/test_doctest/test_doctest.py

```diff
@@ -833,6 +833,121 @@ def test_empty_namespace_package(self):
             self.assertEqual(len(include_empty_finder.find(mod)), 1)
             self.assertEqual(len(exclude_empty_finder.find(mod)), 0)
 
+    def test_lineno_of_test_dict_strings(self):
```

Comment:
As above, lines should be wrapped < 80.  The existing code is mostly consistent about that outside of the places where the doctest text has to be unwrapped.